### ISSUE TITLE
Kernel 4.14

### ISSUE THREAD

**Issue by wchresta** [cransom/surface-pro-3-nixos#2]
Is there an updated version for Kernel 4.14 or is 4.4 still the only stable kernel for SP3?

**Comment by lutzmor** [comment]
Its even possible to use the surface pro with the latest kernel of nixos, i clean up my configs and will add them to github. Currently using the latest kernel with a surface pro 3 and everything seems to work so far :).

**Comment by wchresta** [comment]
Looking forward to it, @lutzmor so far, the wifi adapter always gives me trouble either after a while or after suspend.

**Comment by lutzmor** [comment]
Yeah the suspend i need to test a bit, currently everything seems to work. Redid the system yesterday and will test all the stuff today and see what works and what doesn't :)

**Comment by lutzmor** [comment]
@wchresta are you on the irc of nixos?

**Comment by lutzmor** [comment]
@wchresta i added my config to github now the wifi error only appears when the wireless have power_save on so i currently disabled the power_save option in tlp and i don't have any wifi break downs.

**Comment by lutzmor** [comment]
@wchresta short question was is your normal battery runtime with the old kernel? Mine seems to be quite short ... around 2 - 3 hours and then i need to charge again :/

**Comment by wchresta** [comment]
@lutzmor I have turned on most possible energy optimisations, and wifi is one of the big drains. I just plugged out my laptop with browser and terminal open, and medium screen brightness and the estimation is 5:40 which actually seems about right. If I turn down the screen brightness to low (which is fine if the sun is not shining in my screen), it estimates to 9:40 - I think that's a bit too high but I could probably squeeze out 6 hours.
So yeah, 2-3 seems low.
